tasks/phylogenetic_inference/task_snp_dists.wdl

```diff
@@ -16,13 +17,14 @@ task snp_dists {
   >>>
   output {
     String date = read_string("DATE")
-    String version = read_string("VERSION")
+    String snp_dists_version = read_string("VERSION")
```

Comment:
we're inconsistent across all of the tasks whether we use the `taskname_version` or just `version` in the output. I vote we change them all to be `taskname_version`.